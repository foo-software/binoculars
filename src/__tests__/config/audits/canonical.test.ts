import auditCanonical from '../../../config/audits/canonical';

describe('canonical audit', () => {
  it('should match snapshot', () => {
    expect(auditCanonical('en')).toMatchSnapshot();
  });
});
