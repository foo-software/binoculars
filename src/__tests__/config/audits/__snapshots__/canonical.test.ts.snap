// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`canonical audit should match snapshot 1`] = `
Object {
  "description": "Canonical links suggest which URL to show in search results. [Learn more](https://web.dev/canonical/).",
  "title": "Document has a valid \`rel=canonical\`",
}
`;
